.idea/runConfigurations/go_build_github_com_matcha_devs_matcha.xml

```diff
@@ -3,12 +3,12 @@
     <module name="matcha" />
     <working_directory value="$PROJECT_DIR$" />
     <envs>
-      <env name="MYSQL_PASSWORD" value="" />
+      <env name="MYSQL_PASSWORD" value="xO|I]Br8n}56" />
```

Comment:
This file, again, should not be changed, please.

Reply:
shoot sorry